actions/executeInstructions.ts

```diff
@@ -0,0 +1,54 @@
+import { Proposal, ProposalTransaction } from '@solana/spl-governance'
+
+import { withExecuteTransaction } from '@solana/spl-governance'
+import { RpcContext } from '@solana/spl-governance'
+import { ProgramAccount } from '@solana/spl-governance'
+import { sendSignedTransaction, signTransaction } from '@utils/send'
+
+import { Transaction, TransactionInstruction } from '@solana/web3.js'
+
+// Magic number (?)
+const DEFAULT_TIMEOUT = 31_000
```

Comment:
Max said to remove this timeout I think, mentioned something about having timeout centralized being better

Reply:
copied the TIMEOUT that was used in the original method. We could refer to it, but have to export it. Thus have to mutate common code.

Reply:
I think it's about not passing an explicit timeout at all